src/amo/components/Collection/index.js

```diff
@@ -102,6 +104,24 @@ export type SaveAddonNoteFunc = (
   notes: string,
 ) => void;
 
+// Let's find out what page we'll be on after we remove
+// the item from the collection.
+const getCollectionPage = (collection) => {
```

Comment:
Should it be `getPreviousCollectionPage`?

Reply:
actually here we are trying to figure out the collection page...
let me think about this..

Reply:
ok I think I am going to combine the ceil func inside this function (as you've suggested/asked about below) and so i've changed things a bit...

regarding this name, I decided to go with getNewCollectionPage as it's only called when its determined that we need to switch the page.